<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signin</class>
 <widget class="QDialog" name="signin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Logo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>225</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonMale">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFemale">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Record passphrase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Record Again</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>logo</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>radioButtonFemale</zorder>
  <zorder>radioButtonFemale</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>radioButtonFemale</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>radioButtonMale</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
